lib/ntbb-session.lib.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
 require_once dirname(__FILE__) . '/ntbb-database.lib.php';
+include_once '../../play.pokemonshowdown.com/config.inc.php';
```

Comment:
```suggestion
include_once '../config/config.inc.php';
```

The `config/` is important! Also you don't need to go out and back in, you're already in `play.pokemonshowdown.com`.